Scenes/EncountersMenuScene.gd

```diff
@@ -77,9 +79,9 @@ func _run():
 		addButton("Back", "Close this menu", "endthescene")
 		
 		if(hasSomeoneToForget):
-			addButton("Forget", "Pick which character you wanna forget", "forgetmenu")
+			addButton("NPC list", "Shows NPCs that you encountered", "npclistmenu")
```

Comment:
See how I carefully avoided using the word NPC in my code? x3 They're characters, npc is too game-y

I understand that 'Character list' is too long. Maybe just rename it to 'Memory' then and change all 'NPC' to 'character'